### PR TITLE
feat: add extra commission rate info to genoracle cmd

### LINKED ISSUE #556: Add oracleCommMaxRate, oracleCommMaxChangeRate to add-genesis-oracle cmd

### PR DESCRIPTION
**Pull request by audtlr24**
closes #556.

### CODE REVIEW

File: cmd/panacead/cmd/genoracle.go [hunk 1]
```diff
@@ -156,6 +160,20 @@ func setOracle(cmd *cobra.Command, genState *oracletypes.GenesisState) error {
...
+	if err != nil {
+		return fmt.Errorf("failed to get oracle commission max rate: %w", err)
+	}
+
+	commMaxRate, err := sdk.NewDecFromStr(commMaxRateStr)
+	if err != nil {
+		return fmt.Errorf("inavlid commission max rate: %w", err)
+	}
+
+	if commMaxRate.IsNegative() || commMaxRate.GT(sdk.OneDec()) {
```
Comment (gyuguen) [inline cmd/panacead/cmd/genoracle.go:173]: How about added code like below to line 79? :
Maybe we can replace validation.
```go
if err := oracleGenState.Validate(); err != nil {
	return err
}
```

Comment (gyuguen) [reply cmd/panacead/cmd/genoracle.go:173]: And we need to change `Oracle.ValidationBasic()` like `OracleRegistration.ValidateBasic()`.
```go
if m.OracleCommissionMaxRate.LT(sdk.ZeroDec()) || m.OracleCommissionMaxRate.GT(sdk.OneDec()) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "OracleCommissionMaxRate must be between 0 and 1")
}

if m.OracleCommissionRate.LT(sdk.ZeroDec()) || m.OracleCommissionRate.GT(m.OracleCommissionMaxRate) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "oracleCommissionRate must be between 0 and OracleCommissionMaxRate")
}

if m.OracleCommissionMaxChangeRate.LT(sdk.ZeroDec()) || m.OracleCommissionMaxChangeRate.GT(sdk.OneDec()) {
	return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "OracleCommissionMaxChangeRate must be between 0 and 1")
}
```

**Review by gyuguen — APPROVED** [review]
Thank you.
Please check my comment.

File: cmd/panacead/cmd/genoracle.go (hunk 1 above)
Comment (audtlr24) [reply cmd/panacead/cmd/genoracle.go]: Thanks, I fixed it.

**Review by H4NLee — APPROVED** [review]
lgtm

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Pull request merged**